### PR TITLE
Punish agents for not being on a bike

### PR DESCRIPTION
**Pull request by harryjph**

### CODE REVIEW

File: internal/server/GameLoop.go [hunk 1]
```diff
@@ -267,6 +271,15 @@ func (s *Server) unaliveAgents() {
 	}
 }
 
+func (s *Server) punishBikelessAgents() {
+	for id, agent := range s.GetAgentMap() {
+		if _, ok := s.megaBikeRiders[id]; !ok {
+			// Agent is not on a bike
+			agent.UpdateEnergyLevel(utils.LimboEnergyPenalty)
```
Comment (lucianecchi) [inline internal/server/GameLoop.go:278]: I think this needs a - before the arg (updateenergy adds whatever is passed in if i recall correctly)

**Review by lucianecchi — CHANGES_REQUESTED** [review]

File: internal/server/GameLoop.go (hunk 1 above)
Comment (harryjph) [reply internal/server/GameLoop.go:278]: I added the - in the parameter definition. Would it be preferable to have it here?

Comment (IshaanReni) [reply internal/server/GameLoop.go:278]: LimboEnergy is set to -0.25

Comment (lucianecchi) [reply internal/server/GameLoop.go:278]: sorry I missed that, nah I guess having the word penalty is explicit enough

**Review by lucianecchi — APPROVED** [review]

**Pull request merged**